website/sidebars.js

```diff
@@ -98,6 +98,46 @@ module.exports = {
         },
       ],
     },
+    {
+      type: "category",
+      label: "GraphQL API",
+      collapsible: false,
+      collapsed: false,
+      items: [
+        {
+          type: "doc",
+          label: "Overview",
+          id: "current/api/index",
+        },
+        "current/api/concepts",
+        "current/api/playground",
+        {
+          type: "link",
+          label: "Reference",
```

Comment:
@crjm could you please update #3745 to ensure the static API reference doc is available at this URL? Thanks

Reply:
> @crjm could you please update #3745 to ensure the static API reference doc is available at this URL? Thanks

Sure!